### DISCUSSION TITLE
Function request

### DISCUSSION THREAD

**Discussion by VastOne** [puddletag/puddletag#634] [Q&A] [answered]
Is there a way from Functions / Replace to be able to select wild card? I have several songs that are tagged with things like (Remaster) or (1988) or (Single Version).. What I would like to be able to do is a Function / Replace of (wildcard) with Blank and it takes care of it.. I have tried things like (*) or (*.*) ... if there is something already built in that is great if not it would be a great feature

**Comment by sandrotosi** [comment, marked as answer]
Instead of `Replace`, maybe using `Replace with RegExp` is what you're looking for?
